Formula/dunamai.rb

```diff
@@ -0,0 +1,36 @@
+class Dunamai < Formula
+  include Language::Python::Virtualenv
+
+  desc "Dynamic version generation"
+  homepage "https://pypi.org/project/dunamai/"
```

Comment:
let's use github page rather than the pypi project page
```suggestion
  homepage "https://github.com/mtkennerly/dunamai"
```